internal/bridge/crunchybridgecluster/postgres.go

```diff
@@ -156,7 +156,7 @@ func (r *CrunchyBridgeClusterReconciler) reconcilePostgresRoleSecrets(
 		clusterRole, err := r.NewClient().GetClusterRole(ctx, apiKey, cluster.Status.ID, roleName)
 		// If issue with getting ClusterRole, log error and move on to next role
 		if err != nil {
-			// TODO (dsessler7): Emit event here?
+			// TODO(dsessler7): Emit event here?
```

Comment:
I think we would want this space to be consistent? Not a huge deal but if you end up needing more changes